Package:     squidguard
Subject:     add upstream fix to sgDiv.c
Author:      Joachim Wiedorn <joodebian at joonet.de>
Forwarded:   no
Last-Update: 2015-11-25

Upstream patch number 10 (to sgDiv.c.in) must be used to sgDiv.c, too.
---

diff -urN orig/src/sgDiv.c s00/src/sgDiv.c
--- orig/src/sgDiv.c	2011-10-01 20:49:00.000000000 +0200
+++ s00/src/sgDiv.c	2015-11-25 02:01:30.038527962 +0100
@@ -745,7 +745,7 @@
       p++;
       break;
     case 'u': /* Requested URL */
-      strcat(buf, req->orig);
+      strncat(buf, req->orig, 2048);
       p++;
       break;
     default:
